addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -200,15 +205,7 @@ const Link = Widget.extend({
             this.$link[0].removeAttribute('target');
         }
         if (this._setLinkContent && (data.content !== this.data.originalText || data.url !== this.data.url)) {
```

Comment:
we verify that the content has changed in `_setLinkWithData`, maybe we can simplify?

Reply:
done